build.sbt

```diff
@@ -1350,3 +1353,4 @@ ThisBuild / whitesourceAggregateProjectToken := {
 ThisBuild / whitesourceIgnoredScopes ++= Seq("plugin", "scalafmt", "sxr")
 ThisBuild / whitesourceFailOnError := sys.env.contains("WHITESOURCE_PASSWORD") // fail if pwd is present
 ThisBuild / whitesourceForceCheckAllDependencies := true
+ThisBuild / whitesourceForceCheckAllDependencies := true
```

Comment:
Looks like a duplicated line?

Reply:
Huh. Don't know how that happened. I'll fix that.